galaxy.yml

```diff
@@ -2,7 +2,7 @@
 authors:
   - Ansible Network Community (ansible-network)
 dependencies:
-  "ansible.netcommon": "*"
+  "ansible.netcommon": "--pre"
```

Comment:
This shouldn't even work, does it? `--pre` is a CLI flag, not a version specifier.